### PR TITLE
Add realm writes on main and renderer

### PR DESCRIPTION
**Pull request by mongomoe**
## Description:
This PR adds writes to a realm from both the renderer and the main process

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
+      user: app.currentUser,
+      partitionValue: 'myPartition',
+    },
+  };
+
+  // open a synced realm
+  const realm = await Realm.open(config);
+
+  const dogs = realm.objects('Dog');
+  console.log(`Renderer: Number of Dog objects: ${dogs.length}`);
```
Comment (blagoev) [inline main.js:49]: this should write `Main:`

File: renderer.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const Realm = require('realm');
+const BSON = require('BSON');
+
+const app = new Realm.App({ id: '<Your App ID>' }); // create a new instance of the Realm.App
+
+async function run() {
+  // login with an anonymous credential
+  await app.logIn(new Realm.Credentials.anonymous());
```
Comment (blagoev) [inline renderer.js:8]: login should not be required from the renderer

File: renderer.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const Realm = require('realm');
+const BSON = require('BSON');
+
+const app = new Realm.App({ id: '<Your App ID>' }); // create a new instance of the Realm.App
```
Comment (blagoev) [inline renderer.js:4]: app instance required only if we need to do `login` etc

File: renderer.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+async function run() {
+  // login with an anonymous credential
+  await app.logIn(new Realm.Credentials.anonymous());
+
+  const DogSchema = {
+    name: 'Dog',
+    properties: {
+      _id: 'objectId',
```
Comment (blagoev) [inline renderer.js:13]: I feel the same for objectId as in the other examples. We can simplify it and show this example using `int` as `_id` field. then we don't need to import `BSON` as well. If we have other examples with `objectId` type for the `_id` then maybe we can show that this field can be other types. 
My concern is if all examples show this as type `objectId` developers may assume this is required and start building apps with this type only.

**Review by blagoev — APPROVED** [review]
alright, with some comments

File: main.js (hunk 1 above)
Comment (mongomoe) [reply main.js:49]: 👍

File: renderer.js (hunk 3 above)
Comment (mongomoe) [reply renderer.js:4]: 👍

File: renderer.js (hunk 2 above)
Comment (mongomoe) [reply renderer.js:8]: 👍

File: renderer.js (hunk 4 above)
Comment (mongomoe) [reply renderer.js:13]: Fair point, switching it to `int` now.

**Pull request merged**